### ISSUE TITLE
Missing host argument of connection block?

### ISSUE THREAD

**Issue by gdsotirov** [scarolan/ansible-terraform#5]
Should not the [`connection` block](https://github.com/scarolan/ansible-terraform/blob/291172a7cc6ccca4fc72234a22ca4e5fafd0eef2/terraform_gcp/main.tf#L50) of the remote-exec provisioner have the [required `host` argument](https://www.terraform.io/docs/provisioners/connection.html#argument-reference)? While I was playing with the GCP example, Terraform 0.13 stopped me with the following error:

```
Error: Missing required argument

  on main.tf line 50, in resource "google_compute_instance" "tfansible":
  50: 		connection {

The argument "host" is required, but no definition was found.
```

P.S. I'm using SSH agent, so I removed private key definition.

**Comment by scarolan** [comment]
Yes, it looks like the provider has been updated since this prototype code was created.  this repo is now deprecated so closing this out
